frontend/src/pages/org/archived-item-detail/ui/qa.ts

```diff
@@ -703,7 +703,7 @@ export class ArchivedItemDetailQA extends BtrixElement {
             class="label-same-line"
             label=${msg("Sort by:")}
             size="small"
-            value=${this.qaRunId ? "approved.-1" : "url.1"}
+            value=${this.qaRunId ? "approved.-1" : ".1"}
```

Comment:
Why this change? I think we can/should support `url` still.

Reply:
This sets the default to the new 'Crawl Order' added elsewhere here. The idea is that the QA currently shows pages sorted in alphabetical order, but a better default sort that we're using is seeds first followed by timestamp, so adding that here. the empty string is to indicate that default sort order.

Reply:
Ah okay, thanks for the explanation! Yup that makes sense.